### ISSUE TITLE
Sidebar coloring

### ISSUE THREAD

**Issue by Fire-Dragon-DoL** [tdm00/sublime-theme-railscasts#8]
Considering now it's possible, are there any chance to have sidebar darker? The white sidebar is killing my eyes.

**Comment by tdm00** [comment]
Submit pull request

On Saturday, May 17, 2014, Francesco Belladonna notifications@github.com
wrote:

> Considering now it's possible, are there any chance to have sidebar
> darker? The white sidebar is killing my eyes.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/tdm00/sublime-theme-railscasts/issues/8
> .

## 

_Troy Murray_

**Comment by Fire-Dragon-DoL** [comment]
@tdm00 I already colored the sidebar on my previous sublime text 2 installation, however right now I feel completely lost with sublime text 3, so I definitely need some time, let me play with it.

**Comment by Fire-Dragon-DoL** [comment]
I know lot of time has passed, but I found a great solution to this on Sublime Text 3.
We basically have the support for a theme (coloring of menu/sidebar and such) and color scheme (coloring of the text inside buffers).

Now, instead of writing a custom theme for this, I just used [Soda Theme](https://github.com/buymeasoda/soda-theme) with this **Color Scheme**

My configuration looks like:

```
"color_scheme": "Packages/RailsCasts Colour Scheme/RailsCastsColorScheme.tmTheme",
"theme": "Soda Dark 3.sublime-theme"
```

And the soda sidebar theme fits perfectly.
